fatiando/gridder.py

```diff
@@ -439,3 +444,269 @@ def cut(x, y, scalars, area):
+                    raise ValueError(et)
+                else:
+                    npt.append(npd)
+        else:
+            if _is_integer(npd):
+                raise ValueError(et)
+            elif len(npd) != a.ndim:
+                raise ValueError(et)
+            else:
+                npt = npd
+        for ii in numpy.arange(len(npt)):
+            if npt[ii] <= a.shape[ii]:
+                # How are you supposed to properly assign a multi-line string?
+                et1 = 'gridder.pad_array: Desired padding is less than array '
+                et2 = 'length along dimension '+str(ii)
+                et = et1+et2
+                raise ValueError(et)
+    # Compute numbers to pad on the left and right side of the array along
+    # each dimension
+    nps = []
+    for ii in range(0, nd):
+        nps.append((int(numpy.ceil((npt[ii] - a.shape[ii])/2.)),
+            int(numpy.floor((npt[ii] - a.shape[ii])/2.))))
+
+    # If it will be needed, compute the mean
+    meanneeded = ['lintaper', 'oddreflectiontaper']
+    if padtype.lower() in meanneeded:
+        m = numpy.mean(a)
+    # Use numpy's padding routines where possible
+    if _is_number(padtype):
+        # Pad with value
+        ap = numpy.pad(numpy.copy(a), nps, mode='constant',
+            constant_values = float(padtype))
+    elif padtype.lower() == 'mean':
+        # Pad with the mean
+        ap = numpy.pad(numpy.copy(a), nps, mode='mean')
+    elif padtype.lower() == 'lintaper':
+        # Linearly taper to the mean
+        ap = numpy.pad(numpy.copy(a), nps, mode='linear_ramp', end_values=m)
+    elif padtype.lower() == 'edge':
+        # Pad with edge values
+        ap = numpy.pad(numpy.copy(a), nps, mode='edge')
+    elif padtype.lower() == 'reflection':
+        # Pad with even reflection 
+        ap = numpy.pad(numpy.copy(a), nps, mode='reflect', 
+            reflect_type='even')
+    elif padtype.lower() == 'oddreflection':
+        # Pad with odd reflection
+        ap = numpy.pad(numpy.copy(a), nps, mode='reflect', 
+            reflect_type='odd')
+    elif padtype.lower() == 'oddreflectiontaper':
+        # Pad with odd reflection and a cosine taper to mean
+        ap = (numpy.pad(numpy.copy(a), nps, mode='reflect', 
+            reflect_type='odd') - m)
+        for ii in numpy.arange(nd):
+            ap = numpy.apply_along_axis(_costaper, ii, ap, lp=nps[ii][0],
+                rp=nps[ii][1])
+        ap += m
+    if xy is not None:
+        cp = _padcoords(xy, a.shape, nps)
+    else:
+        # What is the preferred way to do this?
+        cp = None
+
+    return ap, cp, nps
+
+def unpad_array(a, nps):
+    '''
+    Unpads an array using the outputs from pad_array.
+
+    This function takes a padded array and removes the padding from both. 
+    Effectively, this is a complement to gridder.cut for when you already 
+    know the number of elements to remove.
+
+    .. note: Unlike pad_array, this returns a slice of the input array.
```

Comment:
If you replace `Unlike pad_array,` with 

    Unlike :func:`~fatiando.gridder.pad_array`,

Sphinx will replace that with a link the `pad_array` function docs.